### DISCUSSION TITLE
Show code copy button: not working?

### DISCUSSION THREAD

**Discussion by seanmrnda** [adityatelange/hugo-PaperMod#981] [Q&A]
This feature does not seem to work.
All I did was set the `ShowCodeCopyButtons: true`.

**Comment by m1rm** [comment]
I am running on the following config and it works:

config/_default/config.toml
```
[params]
    ...
    ShowCodeCopyButtons = true
```

god.mod
```
go 1.23.5

require (
	github.com/adityatelange/hugo-PaperMod v0.0.0-20250105143455-9f1f414be805 // indirect
)
```

The button "copy" is shown on hover over a code block defined in my markdown with the backticks and when I click it, it changes to "copied" and the code then also is copied to the clipboard
